src/sentry/models/integration.py

```diff
@@ -62,6 +63,35 @@ class Meta:
         unique_together = (("project", "stack_root"),)
 
 
+class ExternalProviders(Enum):
+    GITHUB = 0
+    GITLAB = 1
+
+
+EXTERNAL_PROVIDERS = {
+    ExternalProviders.GITHUB: "github",
+    ExternalProviders.GITLAB: "gitlab",
+}
+
+
+class ExternalTeam(DefaultFieldsModel):
+    __core__ = False
+
+    team = FlexibleForeignKey("sentry.Team")
+    provider = BoundedPositiveIntegerField(
+        choices=(
+            (ExternalProviders.GITHUB, "github"),
+            (ExternalProviders.GITLAB, "gitlab"),
+        ),
+    )
+    external_id = models.TextField()
```

Comment:
Do we need an index on this? What kinds of queries are we likely to use on this table?

Reply:
@wedamija good question! most of the time I expect us to do a join query on `team_id` with the `sentry_team` table. And I expect this table to be read-heavy. 
I think we should go without an index for now. I plan on instrumenting with Sentry Performance, so if it turns out that this is a bottleneck on response times, we can always add an index later.